src/Form/input-classifiers.js

```diff
@@ -116,13 +103,19 @@ const isCCForm = (form) => {
 
 /**
  * Get a CC subtype based on selectors and regexes
- * @param {HTMLInputElement} el
- * @param {HTMLFormElement} form
- * @return {string}
+ * @type (el: HTMLInputElement, form: HTMLFormElement) => string|undefined
  */
 const getCCFieldSubtype = (el, form) =>
     CC_MATCHERS_LIST.find((sel) => checkMatch(el, form, sel))?.type
 
+/**
+ * Get an identities subtype based on selectors and regexes
+ * @type (el: HTMLInputElement, form: HTMLFormElement) => string|undefined
+ */
+const getIDFieldSubtype = (el, form) =>
+    ID_MATCHERS_LIST.find((sel) => checkMatch(el, form, sel))?.type
+// TODO: Is it worth checking all selectors first before moving to labels and stuff?
```

Comment:
Still thinking about this… The advantage is to skip complex checks when web authors have been diligent in use of the proper autocomplete attributes and avoid false matches in those cases (which are a minority).